test/test_autograd.py

```diff
@@ -1624,6 +1624,13 @@ def test_sparse_gather_x_scalar(self):
     def test_sparse_gather_both_scalar(self):
         self._test_sparse_gather((), (), 0)
 
+    def test_to_sparse_autograd(self):
```

Comment:
Why is this function necessary? There's already a to_sparse test in common_method_invocations that autograd should call

Reply:
probably not, I added this first before realizing I should add in common method invocs.

Reply:
I wasn't able to add a sparse mask test in common invocs because it's not setup to handle tests with sparse inputs, so I converted this test to test sparse_mask.

does sparse mask also gets tested indirectly via the common method invocations entry for to_sparse?

Reply:
gradcheck and gradgradcheck are run via the common method invocations, so yes sparse_mask is indirectly tested (in the gradgradcheck step), but it's always good to have an explicit test